async_simple/Executor.h

```diff
@@ -16,11 +16,13 @@
 #ifndef ASYNC_SIMPLE_EXECUTOR_H
 #define ASYNC_SIMPLE_EXECUTOR_H
 
+#include <sys/types.h>
```

Comment:
Why is this needed? And please add new headers in `#ifndef ASYNC_SIMPLE_USE_MODULES`